extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,849 @@
+
+### Buffers and Buffer Views
+
+OUTLINE:
+- Which spec to reference? 3DTILES_binary_buffers or core metadata?
+
+### Morton Order
+
+**[Morton order](https://en.wikipedia.org/wiki/Z-order_curve)** assigns indices to nodes in the same level. The indices are used for lookup in availability buffers.
+
+Using the Morton order serves these purposes:
+
+- Efficient tile location decomposition: The Morton order allows efficient encoding and decoding of locations of a tile in the level to its location in the availability buffers.
+- Efficient traversal: The binary representation of tile locations in the tree level allow for easy traversal of the tileset (finding parent and child nodes).
+- Locality of reference: Adjacent indices are stored close to each other in memory and are close to each other spatially.
+
+#### Morton Order Example
+
+_The following section is non-normative_
+
+The figure below shows the tile location decomposition of the tile `(level, x, y) = (3, 5, 1)`. We first convert the tile location to its Morton index, which is `19`. At Level 3 of a Quadtree, we'll use 6 bits to represent the binary value of the Morton index: `010011`.
+
+![Morton Order](figures/morton-indexing.png)
+
+### Availability Encoding
+
+Availability bitstreams are packed in binary using the format described in the [Boolean Data section](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/specification/Metadata/0.0.0/README.md#boolean-data) of the Cesium 3D Metadata Specification. These bitstreams are subject to alignment requirements described in the [Binary Alignment Rules section](https://github.com/CesiumGS/3d-tiles/tree/3DTILES_binary_buffers/extensions/3DTILES_binary_buffers) of the same specification.
+
+Each availability bitstream must be stored as a separate `bufferView`, but multiple `bufferViews` may be stored in a single `buffer`.
+
+## Tileset JSON
+
+Using implicit tiling, the tileset JSON file describes the root tile, tiling scheme, and template URIs for locating resources.
+
+In the extension object, the following information about the root tile is included:
+
+| Property | Description |
+| ------ | ----------- |
+| `tilingScheme` | Either `QUADTREE` or `OCTREE`|
+| `boundingVolume` | a bounding volume (either a `box` or `region`) describing the root tile |
+| `refine` | Either `ADD` or `REPLACE` as in the Cesium 3D Tiles 1.0 Specification. |
+| `geometricError` | Geometric error of the root tile as described in the Cesium 3D Tiles 1.0 Specification. |
+| `maximumLevel` | Maximum level of the entire tree |
+| `subtreeLevels` | How many distance levels in each subtree. |
+
+Furthermore, template URIs are used for locating subtree JSON files as well as tile contents. The key properties are as follows:
+
+| Option | Description |
+| ------ | ----------- |
+| `subtrees.uri` | template URI for a subtree JSON file. see [Subtrees](#subtrees) for more info |
+| `content.uri` | template URI for the content 3D Models |
```

Comment:
Be careful not to use "3D Models" terminology too much because someday we may support other content types like rasters

```suggestion
| `content.uri` | Template URI for the content |
```